### PR TITLE
fix: boolean settings bug fixed for ftds in checkout and thanks pages

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- boolean settings bug fixed for ftds in checkout and thanks pages

### CODE REVIEW

**Comment by dyersituations** [comment]
@roger-in-kiva Were you able to test locally yet? I know @mcstover has been trying to get a setting saved in dev.

**Comment by roger-in-kiva** [comment]
> @roger-in-kiva Were you able to test locally yet? I know @mcstover has been trying to get a setting saved in dev.

Yeah, @mcstover has added the settings to dev and working correctly with the update.

**Comment by mcstover** [comment]
I want to replicate the failure before merging this. Please hold until we can do some more dev testing in the prior state.

I'm going to try and get the other settings in place to test with config matching what was in prod

File: src/pages/Thanks/ThanksPage.vue [hunk 1]
```diff
@@ -293,10 +294,9 @@ export default {
 		this.isGuest = this.receipt && !data?.my?.userAccount;
 
 		// Enable FTDs message from settings
-		const enableFtdMessageData = data?.general?.ftd_message_enable ?? null;
-		this.enableFtdMessage = enableFtdMessageData ? enableFtdMessageData.value : false;
+		this.isFtdMessageEnable = readBoolSetting(data, 'general.ftd_message_enable.value');
```
Comment (mcstover) [inline src/pages/Thanks/ThanksPage.vue:297]: Is this value now getting set properly to `false` in your local against dev?

Comment (roger-in-kiva) [reply src/pages/Thanks/ThanksPage.vue:297]: Yes, I'm getting `false` locally on this branch.
But also, getting `false` in main branch.

**Review by mcstover — APPROVED** [review]
I've so far been unable to replicate the failure we saw this morning.

All the new settings are in dev now so we can test a little better this go round.

**Pull request merged**